playbook/spec/pb_kits/playbook/pb_currency/currency_spec.rb

```diff
@@ -9,10 +9,12 @@
 
   it { is_expected.to define_enum_prop(:align).with_default("left").with_values("left", "center", "right") }
   it { is_expected.to define_prop(:amount).of_type(Playbook::Props::String) }
+  it { is_expected.to define_prop(:emphasized).of_type(Playbook::Props::Boolean) }
   it { is_expected.to define_prop(:label).with_default("").of_type(Playbook::Props::String) }
   it { is_expected.to define_prop(:unit).of_type(Playbook::Props::String) }
   it { is_expected.to define_enum_prop(:size).with_default("md").with_values("sm", "md", "lg") }
   it { is_expected.to define_prop(:symbol).with_default("$").of_type(Playbook::Props::String) }
+  it { is_expected.to define_enum_prop(:variant).with_default("default").with_values("default", "light", "bold") }
 
```

Comment:
These look great.  Can you just add one classname test (like lines 21-24 of this file)?  That way we can test for the classname outcomes of the new props you added.

Reply:
So the reason that I didn't add classname tests is because I'm not actually changing the classname of the currency kit, but rather the currency wrapper. How would I test a classname of a child element?

Reply:
@bh247484

Reply:
Good call!  Pretty sure there's no way to test for the classname changes for inner divs.  Should be all good.